packages/stashjs/src/auth/profile-store.ts

```diff
@@ -82,7 +84,7 @@ export interface ProfileStore {
   readonly loadDefaultProfile: () => AsyncResult<StashProfile, LoadProfileFailure>
 }
 
-const dir = `${process.env["HOME"]}/.cipherstash`
+const dir = path.resolve(os.homedir(), ".cipherstash")
```

Comment:
Note that the behaviour here isn't 1-1 with StashRS because StashRS allows for specifying a path to use (instead of defaulting the `.cipherstash` in the home dir.

I realise that this change is focused around Windows support, but I wanted to note the difference anyway.

Relevant docs: https://docs.cipherstash.com/reference/client-configuration.html#configpath